secure/json-rpc/src/error.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use hex::FromHexError;
+use std::io;
+use thiserror::Error;
+
+#[derive(Debug, Error, PartialEq)]
+pub enum Error {
+    #[error("JSON RPC call {0} returned a custom internal error: {0}")]
+    InternalRPCError(String, String),
+    #[error("Data does not exist. Missing data: {0}")]
+    MissingData(String),
+    #[error("JSON RPC call {0} failed. Response: {0}")]
+    RPCFailure(String, String),
```

Comment:
Same here, since this adds context that should already be there. We would want to add context outside this error for more information afaiu.

Reply:
See above.